handbook/data.md

```diff
@@ -275,6 +275,45 @@ The following is the description of the air pollution exposure data.
 ```
 `````
 
+### PM2.5 US High Resolution Grid, 2000-2016
+
+`````{dropdown} PM2.5 US Grid
+```{list-table}
+:header-rows: 0
+
+* - spatial_coverage
+  - US
+* - spatial_resolution
+  - 1km x 1km
+* - temporal_coverage
+  - 2000-2016
+* - temporal_resolution
+  - annually
+* - size
+  - ~80 MB/year
+* - fasse_location
+  - `/n/dominici_nsaph_l3/Lab/data/exposure/pm25/whole_us/annual/grid_pts/qd_new_predictions`.
+* - processing_description
+  - Merge by row the 1-column matrix PM2.5 values (`PredictionStep2_Annual_PM25_USGrid_20**0101_20**1231.rds`) with the corresponding 1km x 1km United States Grid Matrix (`USGridSite.rds`).
```

Comment:
Just the `PredictionStep2_Annual_PM25_USGrid_20**0101_20**1231` is merged with `USGridSite.rds ` ?

Reply:
They are not merged, they should be merged. `PredictionStep2_Annual_PM25_USGrid_20**0101_20**1231` alone is just a 1-column matrix without indexes.
Replace ** with the year of interest.